fudy/NoteScreen/StickNoteModel.cpp

```diff
@@ -0,0 +1,199 @@
+#include "StickNoteModel.h"
+
+#include <QDebug>
+
+StickNoteModel::StickNoteModel(QObject* parent)
+    : QAbstractListModel{parent}
+{
+	QSqlQuery query;
+	query.exec("SELECT * FROM fudy");
+
+	while(query.next())
+	{
+		StickItem item;
+		item.id = query.value("id").toInt();
+		item.enable = query.value("enable").toBool();
+		item.header = query.value("header").toString();
+		item.noteText = query.value("noteText").toString();
+		item.visible = query.value("visible").toBool();
+		item.xaxis = query.value("xaxis").toInt();
+		item.yaxis = query.value("yaxis").toInt();
+
+		// Thêm dữ liệu vào QVector
+		m_items.append(item);
+	}
+}
+
+int StickNoteModel::rowCount(const QModelIndex& parent) const
+{
+	if(parent.isValid())
```

Comment:
I feel a little bit confused here

Reply:
If parent is valid, it means we are asking for the number of children under parent, if there are no children under any parent index in this model, we return 0.
If parent is not valid, return the number of top-level items in the model